tests/k8s/k8s-gsg-test.sh

```diff
@@ -12,14 +14,29 @@ function cleanup {
 	kubectl delete -f https://raw.githubusercontent.com/cilium/cilium/master/examples/kubernetes/rbac.yaml
 }
 
-trap cleanup exit
+function gather_logs {
+        mkdir -p ./cilium-files/logs
+        kubectl logs -n kube-system $(kubectl -n kube-system get pods -l k8s-app=cilium | grep -v AGE | awk '{print $1}' ) > ./cilium-files/logs/cilium-logs
+        kubectl logs -n kube-system kube-apiserver-vagrant > ./cilium-files/logs/kube-apiserver
+        kubectl logs -n kube-system kube-controller-manager-vagrant > ./cilium-files/logs/kube-controller-manager-logs
+        journalctl -au kubelet > ./cilium-files/logs/kubelet-logs
+}
+
+function finish_test {
+        gather_files k8s-gsg-test ${TEST_SUITE}
+        gather_logs
+        cleanup
+}
+
+trap finish_test exit
 
 echo "KUBECONFIG: $KUBECONFIG"
 
 cleanup
 
+echo "---- Waiting for cluster to get into a good state ----"
 until [ "$(kubectl get cs | grep -v "STATUS" | grep -c "Healthy")" -eq "3" ]; do 
-	echo "---- Waiting for cluster to get into a good state ----"
+  continue
```

Comment:
the `-n` omits the new line, @tgraf already made this changed but jenkins picks each `.` as a new line so we have a trailing amount of `.`
```
.
.
.
.
.
.
```